src/assets/scripts/client/utilities/windUtilities.js

```diff
@@ -0,0 +1,18 @@
+import { getNormalDistributedNumber } from '../math/core';
+
+export const getNewWind = (data) => {
```

Comment:
another word to avoid, `new`.  that is a reserved word in JS (when used by itself).  a better choice here would be `next`.  so: `calculateNextWind` is, I think, a better name.